src/ucp/core/ucp_context.c

```diff
@@ -121,6 +121,10 @@ static ucs_config_field_t ucp_config_table[] = {
    "Threshold for switching from eager to rendezvous protocol",
    ucs_offsetof(ucp_config_t, ctx.rndv_thresh), UCS_CONFIG_TYPE_MEMUNITS},
 
+  {"MEMTYPE_RNDV_THRESH", "8k",
```

Comment:
IMHO we should support 'auto' which means calculate in UCX as today
and the default should be auto